modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/VerticalTimelineDeprecated.tsx

```diff
@@ -200,7 +200,14 @@ const VerticalTimeline: FC<IVerticalTimelineProps> = ({
 	});
 
 	if (loading) {
-		return <Spinner alignCenter={false} className='flex-grow-1' spacer />;
+		return (
+			<ClayLoadingIndicator
+				className={getCN('flex-grow-1 spinner-root', {
+					['spinner-center']: false,
+					['spinner-spacer']: !false
+				})}
+			/>
```

Comment:
The same:

It should be:

```suggestion
		<ClayLoadingIndicator
			className='flex-grow-1 spinner-spacer spinner-root'
		/>
```